### PR TITLE
Issue 3122: [UX] Separate help text for publishing options

### LINKED ISSUE #3122: [UX] separate help text for publishing options
## Describe your issue or idea

After adding the new scheduled publishing feature in Backdrop 1.10 I noticed that help text for two separate items is grouped together, after all items are listed. 
screen_shot_2018-05-15_at_2 07 02_pm

It would be easier to read and understand if the help text for each item were listed immediately after the item itself.

---
PR by @opi: ~https://github.com/backdrop/backdrop/pull/2195~
PR by @klonos ~https://github.com/backdrop/backdrop/pull/3247~
PR by @jenlampton: ~https://github.com/backdrop/backdrop/pull/3613~
PR by @stpaultim (based upon priors) https://github.com/backdrop/backdrop/pull/3742

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/3122

### CODE REVIEW

File: core/modules/node/node.types.inc [hunk 1]
```diff
@@ -181,16 +181,18 @@ function node_type_form($form, &$form_state, $type = NULL) {
...
-      NODE_NOT_PUBLISHED => t('Draft'),
+      NODE_PUBLISHED => t('Publish immediately'),
+      NODE_NOT_PUBLISHED => t('Save as draft'),
       NODE_SCHEDULED => t('Schedule for later'),
     ),
     '#default_value' => $type->settings['status_default'],
-    '#description' => t('Content saved as draft is only accessible to the content creator or site administrators.') . ' ' . t('Content set to "Schedule for later" will be saved as draft when the content is created, but will be automatically published on the date and time selected.'),
     '#required' => TRUE,
   );
+  $form['workflow']['status_default'][NODE_PUBLISHED]['#description'] = t('Published content can be accessed by all site visitors.');
```
Comment (jenlampton) [inline core/modules/node/node.types.inc:193]: Let's add ...`Anyone who has the "view published content" permission.`

Comment (jenlampton) [inline core/modules/node/node.types.inc:193]: Remove the word "all"

Comment (olafgrabienski) [reply core/modules/node/node.types.inc:193]: In the mentioned line, "all" would be removed from the content type settings but not from the node form.

File: core/modules/node/node.pages.inc [hunk 2]
```diff
@@ -328,18 +328,21 @@ function node_form($form, &$form_state, Node $node) {
...
       NODE_PUBLISHED => $node->nid ? t('Published') : t('Publish now'),
-      NODE_NOT_PUBLISHED => t('Draft'),
-      NODE_SCHEDULED => t('Schedule for later'),
+      NODE_NOT_PUBLISHED => $node->nid ? t('Draft') : t('Save as draft'),
+      NODE_SCHEDULED => $node->nid ? t('Scheduled for later') : t('Schedule for later'),
     ),
     '#default_value' => $status,
-    '#description' => t('Content saved as draft is only accessible to the content creator or site administrators.'),
   );
+  $form['options']['status'][NODE_PUBLISHED]['#description'] = t('All site visitors will be able to access this content.');
```
Comment (jenlampton) [inline core/modules/node/node.pages.inc:339]: Remove this word "All" as well as the other :)

**Comment by jenlampton** [comment]
Replaced by https://github.com/backdrop/backdrop/pull/3613

**Pull request closed without merging**